stories/index.stories.js

```diff
@@ -6,7 +6,9 @@ import { linkTo } from '@storybook/addon-links';
 import { withInfo } from '@storybook/addon-info';
 
 import { ListGroupItem, Input, ListGroup } from 'reactstrap';
-import TreeMenu, { defaultChildren } from '../src/index';
+import TreeMenu, { defaultChildren, ItemComponent } from '../src/index';
+import closedIconImg from '../assets/closedIcon.png';
```

Comment:
Since these images are only used in the story, I'd also move the image files into `stories` folder.